samples/trumpify/trump.py

```diff
@@ -0,0 +1,139 @@
+import os
+import sys
+import threading
+import time
+from datetime import datetime
+from email._parseaddr import mktime_tz, parsedate_tz
+
+from twitter import *
+
+sys.path.append(os.path.abspath('../../'))
+
+from daemo.client import Client
+
+CREDENTIALS_FILE = 'credentials.json'
+
+PROJECT_ID = -1
+REVIEW_PROJECT_ID = -1
+
+TW_CONSUMER_KEY = ''
+TW_CONSUMER_SECRET = ''
+TW_ACCESS_TOKEN = ''
+TW_ACCESS_TOKEN_SECRET = ''
+
+INPUT_TWITTER_NAME = 'HillaryClinton'
+TIMESPAN_MIN = 5
+TWEET_COUNT = 10
+
+auth = OAuth(
+    consumer_key=TW_CONSUMER_KEY,
+    consumer_secret=TW_CONSUMER_SECRET,
+    token=TW_ACCESS_TOKEN,
+    token_secret=TW_ACCESS_TOKEN_SECRET
+)
+
+print "Initializing..."
+twitter = Twitter(auth=auth)
+client = Client(CREDENTIALS_FILE)
+
+
+def to_local_time(tweet_timestamp):
```

Comment:
can you move the Twitter utility functions into a separate module and import them? It would make the code simpler. In particular, `to_local_time`, `is_from_last_interval`, and `fetch_new_tweets` could move. `fetch_new_tweets` could be split into two functions, one in the other module that gets and returns the list `messages_last_interval`, and one here that does the loop and posts the task to daemo